packages/oas/src/converter/parts/headers/Oas3HeadersConverter.ts

```diff
@@ -0,0 +1,78 @@
+import { isObject } from '../../../utils';
+import { LocationParam } from '../LocationParam';
+import { Sampler } from '../Sampler';
+import { UriTemplator } from '../UriTemplator';
+import { HeadersConverter } from './HeadersConverter';
+import { Header, OpenAPIV3 } from '@har-sdk/core';
+
+export class Oas3HeadersConverter extends HeadersConverter<OpenAPIV3.Document> {
+  private readonly uriTemplator = new UriTemplator();
+
+  constructor(spec: OpenAPIV3.Document, sampler: Sampler) {
+    super(spec, sampler);
+  }
+
+  protected createContentTypeHeaders(
+    pathObj: OpenAPIV3.OperationObject
+  ): Header[] {
+    const requestBody = pathObj.requestBody as OpenAPIV3.RequestBodyObject;
+    if (requestBody?.content != null && isObject(requestBody?.content)) {
+      return this.createHeaders(
+        'content-type',
+        Object.keys(requestBody.content)
+      );
+    }
+
+    return [];
+  }
+
+  protected createAcceptHeaders(_pathObj: OpenAPIV3.OperationObject): Header[] {
+    return [];
```

Comment:
I suggest you provide an implementation for this method

```suggestion
    const responses = pathObj.responses as OpenAPIV3.ResponsesObject;
    
    if (isObject(responses)) {
     const code = Math.min(...Object.keys(responses))
    
      return this.createHeaders(
        'accept',
        Object.keys(responses[code].content)
      );
    }

    return [];
```